internal/integration/api/selinux.go

```diff
@@ -82,28 +94,31 @@ func (suite *SELinuxSuite) TestRuntimeFileLabels() {
 		constants.MachineSocketPath:     constants.MachineSocketLabel,
 	}
 
+	// Only running on controlplane
 	expectedLabelsControlPlane := map[string]string{
-		constants.APIRuntimeSocketPath:  constants.APIRuntimeSocketLabel,
-		constants.APISocketPath:         constants.APISocketLabel,
-		constants.DBusClientSocketPath:  constants.DBusClientSocketLabel,
-		constants.UdevRulesPath:         constants.UdevRulesLabel,
-		constants.DBusServiceSocketPath: constants.DBusServiceSocketLabel,
-		constants.MachineSocketPath:     constants.MachineSocketLabel,
-		// Only running on controlplane
-		constants.EtcdPKIPath:                           constants.EtcdPKISELinuxLabel,
+		constants.EtcdPKIPath: constants.EtcdPKISELinuxLabel,
+		// Initially we give it system_u, but further files created by etcd are going to be client_u
+		constants.EtcdDataPath:                          ":object_r:etcd_data_t:s0",
```

Comment:
I should probably have never created `system_u` and `client_u` separately, we do not have nor plan to add any user-based rules, and also stuff like etcd is protected yet runs in the client_u. Will probably merge `client_u` back into `system_u`.